### DISCUSSION TITLE
New directus-sdk

### DISCUSSION THREAD

**Discussion by laportem** [Intevel/nuxt-directus#179] [Q&A] [answered]
Is this package using the new directus-sdk under the hood? 
If not, how is this better than the new directus-sdk.

**Comment by Intevel** [comment, marked as answer]
Hey, basically no one says that this module is better. This module has been around for over a year, way before the new SDK came out. I am considering migrating the module to the new SDK.

Basically, this module has the focus of Nuxt support to communicate with Directus without sacrificing Nuxt 3 features. As does the Strapi module.

As I said I am already considering to use the new SDK, we should track this progress in #173.

**Comment by laportem** [comment]
No worries was just checking before I decided to either use this package or the new SDK
